README.md

```diff
@@ -28,40 +28,11 @@ The Hackathon became a success in just a few days!
 
 # Google Summer of Code, 2022
 
-As announced on the [Linux Foundation Wiki](https://wiki.linuxfoundation.org/gsoc/2022-gsoc-zephyr), this project was selected for the [Google Summer of Code](https://summerofcode.withgoogle.com/) under the umbrella of the [Linux Foundation](https://linuxfoundation.org).
+As announced on the [Linux Foundation Wiki](https://wiki.linuxfoundation.org/gsoc/2022-gsoc-zephyr), this project was selected for the [Google Summer of Code](https://summerofcode.withgoogle.com/) under the umbrella of the [Linux Foundation](https://linuxfoundation.org). View the accepted proposal [here](https://summerofcode.withgoogle.com/proposals/details/CROr49Ia).
 
-## Mission Statement
+The progress of this project is tracked in [issues](https://github.com/zephyrproject-rtos/gsoc-2022-thrift/issues) and visualized with [ganttlab](https://www.ganttlab.com/). Check our weekly update:
```

Comment:
It might be better to point to the project board instead of the issues page.
https://github.com/orgs/zephyrproject-rtos/projects/11